### DISCUSSION TITLE
Add support for multiple state entities for secondary states.

### DISCUSSION THREAD

**Discussion by mihsu81** [jseidl/magic-areas#351] [Ideas / Feature Requests]
@jseidl Would it be possible/feasible to add support for multiple state entities for secondary states, like is the case when excluding entities from being analysed?
At the moment I'm using templates to track the state of multiple entities for a specific secondary state.

I.e. multiple entities to track area accented state.

![image](https://github.com/jseidl/hass-magic_areas/assets/40715497/33664415-c8ad-4da7-8780-e4f32d7d88c2)

Thank you in advance.

**Comment by jseidl** [comment]
Have you tried grouping those sensors and using the group as entity? I'm not 100% confident it would work but worth a try. Otherwise use a template sensor. Having MA track more entities degrades performance a bit, so it's preferred to use Home Assistant constructs (groups/templates) in this case.

**Reply by mihsu81** [reply]
I was already using a template to combine multiple binary sensors.
Thank you for the suggestion to use a group. It works just as well for this purpose and is easier to set up.
If adding additional entities degrades performance, we'll have to continue with groups or templates.
Thanks.
